src/components/DataView/DataView.tsx

```diff
@@ -401,47 +401,53 @@ function DataView (props: DataViewProps): ReactElement  {
 			${props.sticky ? "sticky" : ""}
 		`}
 		>
-			<div className="headerRow title">
-				<DataViewTitleBar
-					title={props.title}
-					buttons={props.buttons}
-					savedViewEnabled={savedViewEnabled}
-					savedView={props.savedView}
-					savedViewState={savedViewState}
-					savedViewCallbacks={savedViewCallbacks}
-					savedViewAllowSharedViewSave={(props.savedViewAllowSharedViewSave !== undefined) ? props.savedViewAllowSharedViewSave : false }
-					loading={props.loading}
-					filter={props.filter}
-					filters={props.filters}
-					activeFilters={props.activeFilters}
-					onActiveFiltersChange={props.onActiveFiltersChange}
-				/>
-			</div>
-			<div className="headerActions">
-				<DataViewActionsRow
-					activeColumnObjs={activeColumnObjs}
-					columns={props.columns}
-					bulkActions={props.bulkActions}
-					checked={state.checked}
-					display={display}
-					displayControlEnabled={displayControlEnabled}
-					displayOptionsFull={displayOptionsFull}
-					limit={props.limit}
-					limitOptions={props.limitOptions}
-					onLimitChange={props.onLimitChange}
-					onDisplayChange={props.onDisplayChange}
-					onSkipChange={props.onSkipChange}
-					skip={props.skip}
-					count={props.count}
-					allColumns={props.columns}
-					onColumnsChange={props.onColumnsChange}
-					onCheckAllClick={onCheckAllClick}
-					onSortChange={props.onSortChange}
-					sort={props.sort}
-					data={props.data}
-					checkedAllPages={state.checkedAllPages}
-				/>
-			</div>
+			{
+				props.title &&

```

Comment:
The DataViewTitleBar doesn't depend 100% on the title. The way this condition is written says "If there's no title then we can't render the component" but DataViews can render the title bar with just buttons, with just active filters, etc, etc (any of the props it receives). What you'd need to do here is evaluate every single prop and not render the component **only** when none are present, if at least 1 prop gets passed then we render it.